### PR TITLE
(IRO-2562) Connected demo data to onboarding.

### PR DESCRIPTION
**Pull request by azhurauski-sc**

### CODE REVIEW

File: src/types/AsyncDataType.ts [hunk 1]
```diff
@@ -4,4 +4,19 @@ export interface AsyncDataProps<T> {
...
+  string,
+  string,
+  string,
+  string,
+  string,
+  string,
+  string,
+  string
+]
+
```
Comment (ndemidovich-sc) [inline src/types/AsyncDataType.ts:21]: Please move it into separate file.

File: src/routes/Onboarding/CreateAccount.tsx [hunk 2]
```diff
@@ -12,32 +12,21 @@ import { FC, useState } from 'react'
...
-  'Grass',
-  'Milk',
-]
+import useCreateAccount from 'Hooks/accounts/useCreateAccount'
+import { MnemonicPhraseType } from 'Types/AsyncDataType'
 
 interface CreateAccountProps {
   desktopMode?: boolean
-  onCreate?: VoidFunction
+  afterCreate?: VoidFunction
```
Comment (ndemidovich-sc) [inline src/routes/Onboarding/CreateAccount.tsx:20]: Previous variant was better

File: src/routes/Onboarding/ImportAccount.tsx [hunk 3]
```diff
@@ -15,20 +15,24 @@ import {
...
 import IconInfo from '@ironfish/ui-kit/dist/svgx/icon-info'
 import BackButtonLink from 'Components/BackButtonLink'
+import useImportAccount from 'Hooks/accounts/useImportAccount'
+import { useNavigate } from 'react-router-dom'
+import { MnemonicPhraseType } from 'Types/AsyncDataType'
 
 interface DesktopModeProps {
   desktopMode?: boolean
-  onImport?: VoidFunction
+  afterImport?: VoidFunction
```
Comment (ndemidovich-sc) [inline src/routes/Onboarding/ImportAccount.tsx:28]: Previous variant was better.

**Review by ndemidovich-sc — CHANGES_REQUESTED** [review]

File: src/routes/Onboarding/CreateAccount.tsx [hunk 4]
```diff
@@ -35,9 +22,11 @@ interface CreateAccountProps {
 
 const CreateAccount: FC<CreateAccountProps> = ({
   desktopMode = true,
-  onCreate,
+  onCreate = () => ({}),
```
Comment (ndemidovich-sc) [inline src/routes/Onboarding/CreateAccount.tsx:25]: Default value is not a VoidFunction. Try to change 
```suggestion
  onCreate = () => {}, // or use noop from lodash
```

File: src/routes/Onboarding/ImportAccount.tsx [hunk 5]
```diff
@@ -153,7 +170,7 @@ const MnemonicPhraseTab: FC<DesktopModeProps> = ({ desktopMode, onImport }) => {
 
 const ImportAccount: FC<DesktopModeProps> = ({
   desktopMode = true,
-  onImport,
+  onImport = () => ({}),
```
Comment (ndemidovich-sc) [inline src/routes/Onboarding/ImportAccount.tsx:173]: Similar as for `onCreate` default value remark.

File: src/routes/Onboarding/CreateAccount.tsx [hunk 6]
```diff
@@ -91,12 +89,16 @@ const CreateAccount: FC<CreateAccountProps> = ({
       <Box>
         <Button
           variant="primary"
-          isDisabled={!saved}
+          isDisabled={!saved || !accountName}
```
Comment (ndemidovich-sc) [inline src/routes/Onboarding/CreateAccount.tsx:92]: Better to move it into separate check function.

**Review by ndemidovich-sc — APPROVED** [review]
Small remarks can be fixed latter.

File: src/routes/Onboarding/CreateAccount.tsx (hunk 4 above)
Comment (azhurauski-sc) [reply src/routes/Onboarding/CreateAccount.tsx:25]: replaced with undefined

**Pull request merged**